gcloud/_apitools/http_wrapper.py

```diff
@@ -223,10 +220,6 @@ def RebuildHttpConnections(http):
                 del http.connections[conn_key]
 
 
-def RethrowExceptionHandler(*unused_args):
-    raise
```

Comment:
Yup.  Once we get everything covered, and can identify just the paths we need in `gcloud.storage` / `gcloud.bigquery`, there will be a bunch more deletion to be done.